gridstatus/tests/test_eia.py

```diff
@@ -72,8 +72,8 @@ def test_list_routes():
 def test_rto_interchange():
     eia = gridstatus.EIA()
 
-    start = "2020-01-01"
-    end = "2020-01-03"
+    start = pd.Timestamp.now() - pd.Timedelta(days=7)
```

Comment:
The reason our tests didn't catch the error was they were using fixed dates. I think it's generally a better idea to use current dates to mimic the data we expect in production today.